_field-types/mapping-parameters/analyzer.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: default
+title: analyzer
+parent: Mapping parameters
+grand_parent: Mapping and field types
+nav_order: 5
+has_children: false
+has_toc: false
+---
+
+# `analyzer`
+
+The `analyzer` mapping parameter is used to define the text analysis process that is applied to a text field during both indexing and searching operations.
+
+The key functions of the `analyzer` mapping parameter are:
+
+1. **Tokenization:** The analyzer determines how the text is broken down into individual tokens (words, numbers) that can be indexed and searched.
```

Comment:
Caveat to call out: Each generated should not be more than 32766 bytes in length as indexing fails otherwise.